### DISCUSSION TITLE
RULES_TTL 支持这样的写法吗？

### DISCUSSION THREAD

**Discussion by Lovest20018** [kkkgo/PaoPaoDNS#221] [问答和帮助] [answered]
因为在一台服务器上有不同的服务，挂的通配符证书，都是子域名。
是否支持这样的写法呢？不同的子域名+端口与同个ip+不同的端口来匹配？
AA.XXX.COM:2888@@192.168.1.2:2888
BBB.XXX.COM:2999@@192.168.1.2:2999

**Comment by kkkgo** [comment, marked as answer]
你这种需求应该用caddy之类的程序来代理分流，DNS并没有端口的说法。
参考 https://github.com/kkkgo/PaoPaoDNS/discussions/204#discussioncomment-10639983

**Reply by Lovest20018** [reply]
ok，。感谢大佬回复。
